int32 request_type
---
string response_message
